src/syscheckd/create_db.c

```diff
@@ -59,8 +59,17 @@ static int read_file(const char *file_name, int opts, OSMatch *restriction)
     if (lstat(file_name, &statbuf) < 0)
 #endif
     {
-        merror("%s: Error accessing '%s'.", ARGV0, file_name);
-        return (-1);
+        if(errno == ENOTDIR){
+		/*Deletion message sending*/
+		char alert_msg[PATH_MAX+4];
+		alert_msg[PATH_MAX + 3] = '\0';
+		snprintf(alert_msg, PATH_MAX + 4, "-1 %s", file_name);
+		send_syscheck_msg(alert_msg);
```

Comment:
This line sends the deletion message. It will only check that the files included in the list provided by the manager still exist. It is similar to real time but for single files.